### DISCUSSION TITLE
Need help: Problems with expose.composite

### DISCUSSION THREAD

**Discussion by Burki24** [Koenkk/zigbee2mqtt#13544] [General]
Hi,

I try to get all additional Informations from 'action' from that device: https://www.zigbee2mqtt.io/devices/511.344.html#iluminize-511.344. But one of these exposes is a composite.

```json
{
"action":"color_move_ep2",
"action_color":{
    "x":0.215,
    "y":0.33
    },
"action_group":61954,
"action_transition_time":0,
"battery":5,"linkquality":123
}
```

So I made the following exposes:

```
exposes.numeric('action_transition_time', ea.STATE),
            exposes.numeric('action_color_temperature', ea.STATE).withUnit('mired'),
            exposes.composite('action_color', 'action_color')
                .withFeature(exposes.numeric('x', ea.STATE))
                .withFeature(exposes.numeric('y', ea.STATE)),      
            ],
```
But 'x' and 'y' will not be filled with needed informations:

![image](https://user-images.githubusercontent.com/20765982/184550765-67474d49-d81d-4703-8a8b-dd5e04cca098.png)

does anyone know, were I make a mistake?

Thanks in advance for your help.

Greetings
Burkhard

**Comment by Koenkk** [comment]
This looks correct to me, can you open an issue in the frontend repo? https://github.com/nurikk/zigbee2mqtt-frontend/issues
